src/net_processing.cpp

```diff
@@ -247,6 +247,7 @@ class PeerManagerImpl final : public PeerManager
 
     /** Implement PeerManager */
     void CheckForStaleTipAndEvictPeers() override;
+    bool FetchBlock(NodeId id, uint256 hash, const CBlockIndex* index) override;
```

Comment:
Maybe document that `index` may be null here?

Reply:
Added function documentation.